modules/printbackends/gtkprintbackendcpdb.c

```diff
@@ -74,12 +74,57 @@ cpdb_get_printer_list(GtkPrintBackend *backend)
   refresh_printer_list(frontendObj);
 }
 
-static void cpdb_printer_get_settings_from_options (GtkPrinter *printer,
-                                                    GtkPrinterOptionSet *options,
-                                                    GtkPrintSettings *settings)
+
+static void
+cpdb_printer_get_settings_from_options (GtkPrinter *printer,
+                                        GtkPrinterOptionSet *options,
+                                        GtkPrintSettings *settings)
 {
+    GtkPrinterOption *option;
+
+    option = gtk_printer_option_set_lookup(options, "gtk-n-up");
+    if (option)
+      gtk_print_settings_set ( settings, GTK_PRINT_SETTINGS_NUMBER_UP, option->value );
+
+    option = gtk_printer_option_set_lookup( options, "gtk-n-up-layout");
+    if (option)
+      gtk_print_settings_set( settings, GTK_PRINT_SETTINGS_NUMBER_UP_LAYOUT, option->value);
+
+    ppd_file_t *ppd_file = gtk_print_cups_get_ppd(GTK_PRINTER_CUPS(printer));
```

Comment:
`gtk_print_cups_get_ppd` is a cups backend method. We should either have a similar method for CPDB or drop it for now and add a TODO and do it later (or even drop it altogether if it got deprecated)